css-map.json

```
@@ -149,7 +149,7 @@
 	"_LKG3z7SnerR0eigPCoK": "lyrics-lyricsContent-isInteractive",
 	"vapgYYF2HMEeLJuOWGq5": "lyrics-lyricsContent-isInteractive",
 	"NiCdLCpp3o2z6nBrayOn": "lyrics-lyricsContent-lyric",
-	"nw6rbs8R08fpPn7RWW2w": "lyrics-lyricsContent-lyric",
+	"BJ1zQ_ReY3QPaS7SW46s": "lyrics-lyricsContent-lyric",
```

then don't replace it but add it.
```suggestion
	"nw6rbs8R08fpPn7RWW2w": "lyrics-lyricsContent-lyric",
	"BJ1zQ_ReY3QPaS7SW46s": "lyrics-lyricsContent-lyric",
```